Corrections (#949)
codemagic-ci-cd/codemagic-docs: content/billing/pricing.md

| Windows VM               | $0.045 / minute                                                                                                                                                 |                                                                                                              |
| Extra build concurrency  | $49 / month                                                                                                                                                     | 

> Consider an annual plan or Enterprise plan if more than three concurrent builds are required or if you would like unlimited build minutes on premium macOS and Linux instances, as well as unlimited team seats. For more information contact us [here](https://codemagic.io/contact/).

Reviewer: ```suggestion
Consider an annual plan or Enterprise plan if more than three concurrent builds are required or if you would like unlimited build minutes on premium macOS, Linux, and Windows instances. For more information contact us [here](https://codemagic.io/contact/).
```

> Consider a Professional plan or Enterprise plan if more than three concurrent builds are required or if you would like unlimited build minutes on premium macOS and Linux instances, as well as unlimited team seats. For more information contact us [here](https://codemagic.io/contact/).

Reviewer: ```suggestion
Consider an annual plan or Enterprise plan if more than three concurrent builds are required or if you would like unlimited build minutes on premium macOS and Linux instances, as well as unlimited team seats. For more information contact us [here](https://codemagic.io/contact/).
```
